src/base/static/components/input-explorer/constants.js

```diff
@@ -1,12 +0,0 @@
-export default {
```

Comment:
I deleted all the `input-explorer` components (used on the VV flavors). They had some Backbone stuff in them that I didn't want to refactor.